static/src/javascripts/projects/common/modules/gallery/lightbox.js

```diff
@@ -0,0 +1,626 @@
+import { getBreakpoint, hasTouchScreen, isBreakpoint } from 'lib/detect';
+import FiniteStateMachine from 'lib/fsm';
+import mediator from 'lib/mediator';
+import template from 'lodash/utilities/template';
+import { supportsPushState, pushUrl, back as urlBack } from 'lib/url';
+import { Component } from 'common/modules/component';
+import { inlineSvg } from 'common/views/svgs';
+import blockSharingTpl from 'raw-loader!common/views/content/block-sharing.html';
+import buttonTpl from 'raw-loader!common/views/content/button.html';
+import endslateTpl from 'raw-loader!common/views/content/endslate.html';
+import loaderTpl from 'raw-loader!common/views/content/loader.html';
+import shareButtonTpl from 'raw-loader!common/views/content/share-button.html';
+import shareButtonMobileTpl from 'raw-loader!common/views/content/share-button-mobile.html';
+import throttle from 'lodash/functions/throttle';
+import { loadCssPromise } from 'lib/load-css-promise';
+
+type ImageJson = {
+    caption: string,
+    credit: string,
+    displayCredit: string,
+    ratio: number,
+    role: string,
+    sizes: string,
+    src: string,
+    srcsets: string,
+};
+
+type GalleryJson = {
+    id: string,
+    headline: string,
+    shouldHideAdverts: boolean,
+    standfirst: string,
+    images: Array<ImageJson>,
+};
+
+const pulseButton = (button: HTMLElement): void => {
+    const $btn = bonzo(button);
+    $btn.addClass('gallery-lightbox__button-pulse');
+    window.setTimeout(() => {
+        $btn.removeClass('gallery-lightbox__button-pulse');
+    }, 75);
+};
+
+class GalleryLightbox {
+    showEndslate: boolean;
+    useSwipe: boolean;
+    swipeThreshold: number;
+    lightboxEl: bonzo;
+    $lightboxEl: bonzo;
+    $indexEl: bonzo;
+    $countEl: bonzo;
+    $contentEl: bonzo;
+    nextBtn: HTMLElement;
+    prevBtn: HTMLElement;
+    closeBtn: HTMLElement;
+    infoBtn: HTMLElement;
+    $swipeContainer: bonzo;
+    handleKeyEvents: Function;
+    resize: Function;
+    toggleInfo: Function;
+    fsm: FiniteStateMachine;
+    states: Object;
+    images: Array<ImageJson>;
+    swipeContainerWidth: number;
+    $slides: bonzo;
+    index: number;
+    $images: bonzo;
+    galleryJson: GalleryJson;
+    bodyScrollPosition: number;
+    endslateEl: bonzo;
+    endslate: Object;
+
+    constructor() {
+        // CONFIG
+        this.showEndslate =
+            getBreakpoint() !== 'mobile' &&
+            config.get('page.section') !== 'childrens-books-site' &&
+            config.get('page.contentType') === 'Gallery';
+        this.useSwipe = hasTouchScreen();
+        this.swipeThreshold = 0.05;
+
+        // TEMPLATE
+        const generateButtonHTML = label => {
+            const tmpl = buttonTpl;
+            return template(tmpl, {
+                label,
+            });
+        };
+
+        const galleryLightboxHtml: string =
+            `${'<div class="overlay gallery-lightbox gallery-lightbox--closed gallery-lightbox--hover">' +
```

Comment:
Since this a template literal you don't have to use concatenation and just use it as a multiline string.